### PR TITLE
Verdictd: download rats-tls by gitclone

### PR DESCRIPTION
**Pull request by liangzhou121**
Signed-off-by: Zhou Liang <liang.a.zhou@linux.alibaba.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,7 @@ Eaa_Name := eaa
 Verdict_Name := verdict
 Verdictd_Name := verdictd
 Opa_Name := libopa.so
+RatsTLS_srcdir := $(CUR_DIR)/rats-tls
```
Comment (jiazhang0) [inline Makefile]: Use Rats_Tls_Src_Dir instead, which suits for your variable naming style in this Makefile.

File: Makefile [hunk 2]
```diff
@@ -27,10 +28,11 @@ all: $(Eaa_Name)
 opa:
 	@cd $(Opa_Lib_Path) && go build -o $(Opa_Name) -buildmode=c-shared $(Opa_SRC_Files)
 
-rats-tls:
-	cd $(PROJDIR)/rats-tls && cmake -DBUILD_SAMPLES=on -H. -Bbuild && make -C build install
+RatsTLS:
```
Comment (jiazhang0) [inline Makefile]: rename to "build_rats_tls".

File: Makefile [hunk 3]
```diff
@@ -18,6 +18,7 @@ Eaa_Name := eaa
 Verdict_Name := verdict
 Verdictd_Name := verdictd
 Opa_Name := libopa.so
+RatsTLS_srcdir := $(CUR_DIR)/rats-tls
 
 .PHONY: all
```
Comment (jiazhang0) [inline Makefile]: Add "build_rats_tls" to .PHONY.

**Review by jiazhang0 — CHANGES_REQUESTED** [review]

File: Makefile [hunk 4]
```diff
@@ -56,5 +58,5 @@ package:
 clean:
 	@rm -rf $(Opa_Lib_Path)/libopa.*
 	cargo clean && rm -f Cargo.lock
-	cd $(PROJDIR)/rats-tls && make -C build clean
+	@rm -fr $(RatsTLS_srcdir)
```
Comment (haosanzi) [inline Makefile]: The `/usr/share/rats-tls/` and `/usr/local/lib/rats-tls` also should be uninstalled

**Review by haosanzi — APPROVED** [review]

File: Makefile [hunk 5]
```diff
@@ -54,7 +56,8 @@ package:
...
 clean:
-	@rm -rf $(Opa_Lib_Path)/libopa.*
+	cd $(Rats_Tls_Src_Dir) && make -C build uninstall
 	cargo clean && rm -f Cargo.lock
-	cd $(PROJDIR)/rats-tls && make -C build clean
-	@rm -f dist/rpm/verdictd.spec dist/deb/debian/changelog
\ No newline at end of file
+	@rm -rf $(Opa_Lib_Path)/libopa.*
+	@rm -f dist/rpm/verdictd.spec dist/deb/debian/changelog
+	@rm -fr $(Rats_Tls_Src_Dir)
```
Comment (jiazhang0) [inline Makefile]: I think you can preserve the source code of rats-tls. Instead, use distclean rule to delete rats-tls:

```
.PHONY: all clean distclean build_rats_tls

distclean:
   $(MAKE) clean
   @rm -fr $(Rats_Tls_Src_Dir) 
```

File: Makefile [hunk 6]
```diff
@@ -18,19 +18,21 @@ Eaa_Name := eaa
 Verdict_Name := verdict
 Verdictd_Name := verdictd
 Opa_Name := libopa.so
+Rats_Tls_Src_Dir := $(CUR_DIR)/rats-tls
 
-.PHONY: all
+.PHONY: all build_rats_tls distclean
```
Comment (jiazhang0) [inline Makefile:23]: Forget to add `clean`.

**Pull request merged**